index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <script src="https://kit.fontawesome.com/70b9059057.js" crossorigin="anonymous"></script>
+  <link rel="preconnect" href="https://fonts.googleapis.com"><link rel="preconnect" href="https://fonts.gstatic.com" crossorigin><link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="./index.css">
+  <title>Personal Portfolio</title>
+</head>
+<body>
+  <div class="wrapper">
+    <header class="tool-bar">
+      <a href="#" class="logo"><h2>John Doe</h2></a>
+      <nav>
+        <i class="fa-solid fa-bars fa-2x"></i>
+      </nav>
+    </header>
+    <section class="headline">
+      <hgroup>
+        <h1 class="head-main-title">Hey There. I'm Alex</h1>
+        <h3 class="head-main-sub-title">I’am a Software Developer</h3>
```

Comment:
- [optional] Good job making the changes as requested to the heading title, But I believe now it's a bit smaller, Please consider changing and using the appropriate size that would go with the Figma design.
- [Optional] there is a white margin surrounding your design, As you can see from the Figma design there are no white margin's from left, right, top, and bottom. Please use add `margin: 0` and `padding: 0` to tour `body`.

### Figma design 
![Screenshot 2023-01-13 214729](https://user-images.githubusercontent.com/69077061/212396703-532f6e09-fa30-4a3a-9a9d-c3e5421efdd4.png)

### Your design 
![Screenshot 2023-01-13 214714](https://user-images.githubusercontent.com/69077061/212396464-ef8174dd-ae87-4f04-9646-353c165f1cc6.png)